config/initializers/scrna_core_cell_extraction_config.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'csv'
+scrna_core_cell_extraction_pooling_csv =
```

Comment:
Is this bit irrelevant now? Did there used to be a file called `config/scrna_core_cell_extraction_pooling.csv` that you've now removed?